_posts/JayYang95/2016-02-03-Chapter5Loops.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+author: JayYang95
+title: "Jay's Chapter 5 Loops Exercises"
+---
+
+Exercise 1:
+<iframe src="https://trinket.io/embed/python/79a3635760" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+This exercise was rather simple. I kept track of the count in a variable that incremented by 1 everytime the user entered an integer and added that integer to the overall sum. The average was calculated at the end by dividing the total sum by total count.
```

Comment:
Nice.

One thing to pay attention to is what happens when you set `yes = False`.  `break` will immediately exit.  a quick example we'll discuss in class: https://trinket.io/python/44b5d714de